File: c files/Modulus.c
#include <stdio.h>

int main() {
    int dividend, divisor, remainder;
    float quotient;
    
    printf("Enter number to be divided\n");
    scanf("%d", &dividend);
    
    printf("Enter the divisor\n");
    scanf("%d", &divisor);
    
    quotient = (float)dividend / divisor;
    remainder = dividend % divisor;
    
    printf("The remainder is %d\n", remainder);
    
    return 0;
}

File: py files/Modulus.py
print("Enter number to be divided")

# Get dividend
dividend = int(input())
print("Enter the divisor")

# Get divisor
divisor = int(input())
quotient = float(dividend) // divisor

# Quotient will be calculated
remainder = dividend - divisor * quotient
print("The remainder is " + str(remainder))

# Remainder will be displayed
